mor/mor/feel/feelmor/toolboxmor.hpp

```diff
@@ -17,6 +17,65 @@ makeToolboxMorOptions();
 FEELPP_EXPORT AboutData
 makeToolboxMorAbout( std::string const& str = "opusheat-tb" );
 
+template<typename MeshType>
+class ToolboxMorModelBase
```

Comment:
add documentation for the class
use doxygen mode from vscode to help with this

Reply:
Ok, but I think we should also rename ToolboxMor to something like DEIMMor or something. It is not really related to the toolboxes, but more to DEIM, we could use it without toolboxes.
What do you think ?